referenceframe/model.go

```diff
@@ -160,7 +160,7 @@ func (m *SimpleModel) DoF() []Limit {
 	}
 	m.lock.RUnlock()
 
-	limits := make([]Limit, 0, len(m.OrdTransforms)-1)
+	limits := make([]Limit, 0, len(m.OrdTransforms))
```

Comment:
fixed in https://github.com/viamrobotics/rdk/commit/5f9414a0f8bcfbe1c2128dec4b65b9ff4ff0a1b5